src/components/BoxVaultAPY/BoxVaultAPY.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { MouseEvent, ReactElement } from "react";
+import Token from "@yearn/yearn-web-assets";
+import styled, { css } from "styled-components";
+import { Heading1, Text } from "../../typography";
+import { Icon } from "..";
+import { IconDirection } from "../Icon/Direction";
+
+export interface BoxVaultAPYProps {
+  address: string;
+  chainID?: "1" | "250" | "42161";
+  imgFormat?: "img" | "svg";
+  label: string;
+  value: string;
+  onClick?: (event: MouseEvent<HTMLButtonElement>) => void;
+}
+
+export const BoxVaultAPY = ({
+  address,
+  chainID,
+  label,
+  value,
+  imgFormat,
+  ...props
+}: BoxVaultAPYProps): ReactElement => (
+  <StyledBox {...props}>
+    <StyledCol>
+      <div style={{ marginRight: "1rem" }}>
+        <Token
+          format={imgFormat ?? "svg"}
+          address={address}
+          chainID={chainID ?? "1"}
```

Comment:
Should we be defaulting to chain id 1? Or return `null` if that's not present?

Reply:
I would rather have a default to 1